src/physics/large_eddy_simulation.h

```diff
@@ -0,0 +1,98 @@
+#ifndef __NAVIER_STOKES__
+#define __LARGE_EDDY_SIMULATION__
+
+#include "navier_stokes.h"
+
+namespace PHiLiP {
+namespace Physics {
+
+/// Large Eddy Simulation equations. Derived from Navier-Stokes for modifying the stress tensor and heat flux, which is derived from PhysicsBase. 
+template <int dim, int nstate, typename real>
+class LargeEddySimulationBase : public NavierStokes <dim, nstate, real>
+{
+public:
+	/// Constructor
+	LargeEddySimulationBase( 
+	    const double                                              ref_length,
+	    const double                                              gamma_gas,
+	    const double                                              mach_inf,
+	    const double                                              angle_of_attack,
+	    const double                                              side_slip_angle,
+	    const double                                              prandtl_number,
+        const double                                              reynolds_number_inf,
```

Comment:
```suggestion
          const double                                              reynolds_number_inf,
```
see if that fixes spacing issues.